lib/bango/forms.py

```diff
@@ -213,7 +174,7 @@ class CreateBillingConfigurationForm(SellerProductForm):
 
     @property
     def bango_data(self):
```

Comment:
I added some comments at the top, I tried to leave forms about Bango data and serializers about solitude data. That's for a couple of reasons, but an API that contains an object (eg: `seller_bango`) and so form data are confusing as heck to DRF. Also I was finding that the churn was getting horrendous on this pull request, the more I could leave in the form, the less changes that occurred in the pull request.